web/src/main/webapp/WEB-INF/pages/candidate_create.jsp

```diff
@@ -56,237 +59,235 @@
-                                      value="${fn:escapeXml(candidate.name)}"
+                         <div class="row">
+                            <div class="form-group col-lg-2 col-md-2 col-sm-2 text-right" id="nameLabelDiv">
+                              <spring:message code="candidate.name.field" var="i18nname"/>
+                              <label class="control-label" for="name">
+                                <spring:message code="candidate.name.label"/>
+                              </label>
+                            </div>
+                            <div class="form-group col-lg-4 col-md-4 col-sm-4 labels" id="nameValueDiv">
+                                <spring:message code="candidate.error.name.empty" var="nameEmptyValue"/>
+                                    <input type="text" class="input form-control" name="name" id="name" data-bind="valueWithInit: 'name'"
+                                       value="${fn:escapeXml(candidate.name)}"
                                        placeholder="${i18nname}"
                                        data-error="${nameEmptyValue}"
                                        required maxlength="100">
 
-                                <div id="name-errors" class="help-block with-errors"></div>
-                                <p class="showValue form-control-static"><c:out value="${candidate.name}"/></p>
-                            </div>
-
-                        </div>
-                        <div class="form-group"
-                             id="placeDiv">
-                            <spring:message code="candidate.place.field" var="i18nplace"/>
-                            <label class="control-label col-lg-2 col-md-2 col-sm-2 text-right"
-                                   for="referer"><spring:message
-                                    code="candidate.place.label"/></label>
-
-                            <div class="form-group col-lg-4 col-md-4 col-sm-4">
-                                <input type="text" class="input form-control" name="referer" id="referer" data-bind="valueWithInit: 'referer'"
-                                       value="${fn:escapeXml(candidate.referer)}"
-                                       placeholder="${i18nplace}" maxlength="20">
-
-                                <p class="showValue form-control-static"><c:out value = "${candidate.referer}"/></p>
+                                  <p class="showValue"><c:out value="${candidate.name}"/></p>
+                                  <div id="name-errors" class="help-block with-errors"></div>
                             </div>
-
-                        </div>
-                        <div class="error col-lg-12 col-md-12 col-sm-12">
-                        </div>
-                        <div class="form-group"
-                             id="emailDiv">
+                            <div class="form-group col-lg-2 col-md-2 col-sm-2" id="placeLabelDiv">
```

Comment:
You can use this example as a base for building up your multi-column layout: http://www.bootply.com/s2mmi1YyL4